src/elements/accordion/src/index.tsx

```diff
@@ -0,0 +1,67 @@
+/** @jsx jsx */
+
+import React, { useState } from 'react'
+import { jsx, useThemeUI } from 'theme-ui'
+import { Icon } from '@uswitch/trustyle.icon'
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  title: string
+  isInitiallyOpen?: boolean
+}
+
+const Accordion: React.FC<Props> = ({
+  title,
+  isInitiallyOpen = false,
+  children
+}) => {
+  const {
+    theme: { accordion: accordionTheme = {}, colors = {} }
+  }: any = useThemeUI()
+  const [isOpen, setIsOpen] = useState(isInitiallyOpen)
+
+  return (
+    <div>
+      <button
+        sx={{
+          cursor: 'pointer',
+          border: 0,
+          display: 'flex',
+          flexDirection: 'row',
+          width: '100%',
+          ...accordionTheme.button,
+          ...(isOpen ? accordionTheme.isActive.button : {})
+        }}
+        onClick={() => setIsOpen(!isOpen)}
+      >
+        <div
+          sx={{
+            flex: '1',
+            textAlign: 'left'
+          }}
+        >
+          {title}
+        </div>
+        <Icon
+          color={
+            isOpen
+              ? colors[accordionTheme.isActive.caret.color]
```

Comment:
this logic slightly differs from the logic used by theme-ui as it doesn't fall back to `accordionTheme.isActive.caret.color`. You can either add it manually, or you can use the workaround I used in the pagination component: https://github.com/uswitch/trustyle/blob/master/src/elements/pagination/src/index.tsx#L66

Also I'm guessing you're waiting for the optional chaining PR before making sure this code won't error if `isActive` isn't defined?

Reply:
Yeah I thought of having it fallback to the value passed if a match in `colors` isn't found but I left it as really it should always use a theme colour.

And yeah, I will patch with optional chaining once that's in